components/scream/src/interface/CMakeLists.txt

```diff
@@ -10,8 +13,10 @@ else()
 endif()
 
 set(CSM_SHARE "${INSTALL_SHAREDPATH}/${COMP_INTERFACE}/${ESMFDIR}/${NINST_VALUE}/csm_share")
+link_directories(${SCREAM_TPL_LIBRARY_DIRS})
 
 add_library(atm ${ATM_SRC})
-target_include_directories(atm PUBLIC ${SCREAM_INCLUDE_DIRS} ${SCREAM_TPL_INCLUDE_DIRS} ${CSM_SHARE} ${INSTALL_SHAREDPATH}/include)
+target_include_directories(atm PUBLIC ${SCREAM_INCLUDE_DIRS} ${SCREAM_TPL_INCLUDE_DIRS} ${CSM_SHARE} ${INSTALL_SHAREDPATH}/include ${CATCH_INCLUDE_DIR})
```

Comment:
Why is catch needed for a library?

Reply:
Not sure, I think I copied this line from somewhere else.  I will remove the catch.

Reply:
I think it's because you were including catch in `scream_cxx_f90_interface.cpp `. Now that you removed it, you can revert this change.